<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Google Translate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 233, 255);</string>
  </property>
  <widget class="QLineEdit" name="tb_text1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="english_to_persian">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>English To Persian</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="persian_to_english">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Persian To English</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_text_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;

</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_translate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 135, 79);</string>
   </property>
   <property name="text">
    <string>Translate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
